### ISSUE TITLE
Lacking CONTRIBUTING.md

### ISSUE THREAD

**Issue by Sachin-Bhat** [langchain-ai/langchain-elastic#36]
Hello everyone,

I believe that the project currently lacks instructions on contributing to the project. Would greatly appreciate if there are instructions on setting up poetry and the test environment. This can also help new contributors write and see if their tests work.

Cheers,
Sachin

**Comment by miguelgrinberg** [comment]
Yes, we should add this information here. Note that most of the supporting code for this integration is in the [Elasticsearch client library](https://github.com/elastic/elasticsearch-py), which does have information for contributors. We'll update this repository with a version of that adapted to this project. Thanks!
